src/components/Navlogo.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState, useEffect } from 'react';
+import './Navlogo.scss';
+const NavLogo = () => {
+  const logoText = 'Kurly\nFlower';
+  const [text, settext] = useState('');
+  const [count, setcount] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      settext(text + logoText[count]); //이전에 set으로 받아온것 + 다음에 받아올것
+      setcount(count + 1); // 글자가 1글자씩 나오게 함.
```

Comment:
```suggestion
      setcount(prevCount => prevCount + 1); // 글자가 1글자씩 나오게 함.
```
지금은 이펙트에 의존성 배열이 없기 때문에 문제가 생기지 않고 있는데 의미에 맞게 사용하려면 의존성 배열에 빈배열을 넣어주어야 하고, 그렇게 되면 count가 예전 값을 바라보는 문제가 생길 것 같습니다. 이 문제를 벗어나려면 함수를 사용하여 setState 해주면 됩니다.